.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,203 @@
+name: CICD Pipeline
+
+on:
+  push:
+    branches:
+      - master
+      - VPS-149/refactoring-github-actions-workflow
```

Comment:
Is the inclusion of the `VPS-149/refactoring-github-actions-workflow` branch intentional?

Reply:
on the fence, on one hand it'll probably be useful for future reference, if anyone new to CICD wants to edit github actions it might make a bit more sense, plus there isn't really any harm done...
on the other hand it probably doesn't belong in production code

so should I remove it?

Reply:
I reckon remove it - if they are editing Github actions configs, they should _(hopefully)_ know what they are doing already. We do also still have the master branch on there so hopefully it won't be too hard to understand?

Reply:
branch name removed from on push